.babelrc

```diff
@@ -1 +1,4 @@
-{ "presets": ["es2015"] }
+{
+  "presets": ["es2015"],
+  "plugins": ["add-module-exports"]
```

Comment:
Indeed, unless use `import xxx from 'xxx'` instead of `require`. Do we consider to change to new syntax??